### PR TITLE
DummyWireComponent

### PR DESCRIPTION
**Pull request by collinlunn**
Spawns fake cosmetic wires under machines so that real wires appear to tile to the machine.

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Power/wires.yml [hunk 1]
```diff
@@ -138,3 +138,69 @@
...
+  parent: BaseDummyWire
+  id: HVDummyWire
+  name: HV Connector Wire
+  components:
+  - type: Sprite
+    sprite: Constructible/Power/hv_cable.rsi
+    state: hvcable_0
+  - type: Icon
+    sprite: Constructible/Power/hv_cable.rsi    
+    state: hvcable_4
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Constructible/Power/wires.yml:169]: Are these iconcomponents needed?

File: Content.Server/GameObjects/Components/Power/DummyWireComponent.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Serialization;
+using System.Collections.Generic;
+
+namespace Content.Server.GameObjects.Components.Power
+{
+    [RegisterComponent]
+    public class DummyWireComponent : Component, IMapInit
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Power/DummyWireComponent.cs:11]: Component should be client-side if possible IMO.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Client/GameObjects/Components/Power/DummyWireComponent.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        public override string Name => "DummyWire";
+
+        private List<string> _dummyWireProtos = default!;
+
+        private List<IEntity> _dummyWires = new();
+
+        public override void ExposeData(ObjectSerializer serializer)
+        {
+            base.ExposeData(serializer);
+            serializer.DataField(ref _dummyWireProtos, "prototype", new List<string> { "HVDummyWire" });
```
Comment (PaulRitter) [inline Content.Client/GameObjects/Components/Power/DummyWireComponent.cs:21]: if its a list i'd call it prototypes
```suggestion
            serializer.DataField(ref _dummyWireProtos, "prototypes", new List<string> { "HVDummyWire" });
```

**Review by PaulRitter — CHANGES_REQUESTED** [review]
since all the dummywirecomponent does is spawn a prototype underneath the entity, i'd argue on maybe naming it a bit more generic, maybe SpawnPrototypeComponent

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**